http-client/src/main/java/io/airlift/http/client/jetty/JettyResponse.java

```diff
@@ -31,6 +32,16 @@ public int getStatusCode()
         return response.getStatus();
     }
 
+    @Override
+    public HttpVersion getHttpVersion()
+    {
+        return switch (response.getVersion()) {
+            case HTTP_1_1 -> HttpVersion.HTTP_1_1;
```

Comment:
We could map `HTTP_0_9` and `HTTP_1_0` to `HTTP_1_1`. Or maybe name ours `HTTP_1` since we don't care about the distinction?